server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -92,9 +92,14 @@ grant codeBase "${codebase.reactor-core}" {
   permission java.net.SocketPermission "*", "connect,resolve";
 };
 
+grant codeBase "${codebase.opensearch-arrow-memory-shaded}" {
```

Comment:
This is not the right permissions configuration

Reply:
do you mean the syntax? It seems like it worked, but let me double check.

Reply:
`permission java.security.AllPermission` - implies all other permission, should never be granted

Reply:
my bad! let me fix it

Reply:
done